src/modules/list.marko

```diff
@@ -230,130 +297,449 @@
+                                    Item 2
+                                </span>
+                                <span class="list__trailing">
+                                    <svg
+                                        aria-hidden="true"
+                                        class="icon icon--16"
+                                        height="16"
+                                        width="16"
+                                    >
+                                        <icon-symbol name="money-back-guarantee-16"/>
+                                    </svg>
+                                </span>
+                            </div>
+                        </li>
+                    </ul>
+                </div>
+            </div>
+        </div>
+    </div>
+
+    <highlight-code type="html">
+        <div class="list">
+            <ul>
+                <li>
+                    <div>
+                        <span class="list__leading">
+                            <svg
+                                aria-hidden="true"
+                                class="icon icon--16"
+                                height="16"
+                                width="16"
+                            >
+                                <use href="#icon-file-16"/>
+                            </svg>
+                        </span>
+                        <span class="list__body">
+                            Item 1
+                        </span>
+                    </div>
+                </li>
+                <li>
+                    <div>
+                        <span class="list__leading">
+                            <svg
+                                aria-hidden="true"
+                                class="icon icon--16"
+                                height="16"
+                                width="16"
+                            >
+                                <use href="#icon-file-16"/>
+                            </svg>
+                        </span>
+                        <span class="list__body">
+                            Item 2
+                        </span>
+                        <span class="list__trailing">
+                            <svg
+                                aria-hidden="true"
+                                class="icon icon--16"
+                                height="16"
+                                width="16"
+                            >
+                                <use href="#icon-money-back-guarantee-16"/>
+                            </svg>
+                        </span>
+                    </div>
+                </li>
+            </ul>
+        </div>
+    </highlight-code>
+    <h3 id="list-default">
+        List with fieldset
+    </h3>
+    <p>
+        For an interactive list, you can add a fieldset with radios or checkboxes
+    </p>
+    <div class="demo">
+        <div class="demo__inner">
             <div class="list">
```

Comment:
It would be ideal for the general markup to be the same between the variations. Can we not use `<ul>` here like we do in other variations?

Reply:
Can we do both? I'd hate to add an extra layer, but I believe there may be an `a11y` benefit.

Reply:
Both should be supported out of the box. Just need to change the leading to contain the radio. 
I don't think we need to document it for now.